### ISSUE TITLE
csm_db_history_archive.py failed ImportError: No module named csm_db_rollup

### ISSUE THREAD

**Issue by thanh-lam** [IBM/CAST#729]
**Describe the bug**
Command `csm_db_history_archive.py` failed:
```
[root@f4n05 db]# ./csm_db_history_archive.py
Traceback (most recent call last):
  File "./csm_db_history_archive.py", line 39, in <module>
    from csm_db_rollup import  rollupDir
ImportError: No module named csm_db_rollup
```

**To Reproduce**
1. Go to `/opt/ibm/csm/db/` and run `csm_db_history_archive.py`
2. See error:
```
[root@f4n05 db]# ./csm_db_history_archive.py
Traceback (most recent call last):
  File "./csm_db_history_archive.py", line 39, in <module>
    from csm_db_rollup import  rollupDir
ImportError: No module named csm_db_rollup
```

**Expected behavior**
Command should not return error.

**Screenshots**

**Environment (please complete the following information):**
 - Machine [IST alt. cluster master f4n05]
 - Version [e.g. CSM 1.6.0-2599]

**Additional context**
Command worked fine at previous level: 1.6.0-2583.

**Issue Source:**
IST CSM testing for GA.

**Comment by williammorrison2** [comment]
@mew2057 @thanh-lam I think by adding the csm_db_rollup.py to the CAST/csmdb/sql/ 	CMakeLists.txt file should address the problem.

**Comment by mew2057** [comment]
This is a missing python file in the CMake. I'll add it in the next few moments.

**Comment by mew2057** [comment]
Fix is in 1.6.x release branch.

**Comment by thanh-lam** [comment]
Verified that issue is fixed with CSM build 1.6.0-2600.
